### ISSUE TITLE
Add note to deployment docs about how to change the website title

### ISSUE THREAD

**Issue by liamoreilly** [eecs-autograder/ag-website-vue#519]
The website title (both the HTML title and the page 'title') are currently "Autograder.io". This might be confusing for deployments and the main autograder.io site. Previously to commit [6caef99768c1e1404b9023552f870e259bec07d9](https://github.com/eecs-autograder/ag-website-vue/commit/6caef99768c1e1404b9023552f870e259bec07d9) it was just Autograder.

The commit [6caef99768c1e1404b9023552f870e259bec07d9](https://github.com/eecs-autograder/ag-website-vue/commit/6caef99768c1e1404b9023552f870e259bec07d9) seems to do something with true/mutant names. Is it possible to customise the name? If so, the documentation does not mention this(as far as I can tell). If not, then should we consider this?

Thanks,
Liam

**Comment by james-perretta** [comment]
@liamoreilly It's certainly possible to change the page title on your deployment by just editing the html in that file. Doing so shouldn't cause any issues since it's just a cosmetic change.

The thing that makes it tricky just from user standpoint is the branding of Autograder.io being the same as the domain name. I'm not sure what the right solution for that is, but you can certainly change it to whatever you like on your deployment. Does that help?

**Comment by liamoreilly** [comment]
@james-perretta Sorry, I have no idea why I didn't think of that. I assumed somehow that the mutant stuff in the commit was meant to morph the title.

Changing the code base is generally not so nice for configuration elements, but this works just fine. I would just say it doesn't scale well. But I am not sure there would be much more to change the names of anyway.

We refer to our deployment as csautograder as that is the start of our local URL (i.e., the server name). We haven't found any confusion with your main deployment. But I thought if the branding (titles) on our deployment is now Autograder.io then this may cause some confusion. Previously the title was Autograder. I have now changed it to CSAutograder which is actually a bit more clear that it is the departmental deployment and not your main deployment. Overall, I see no issues with the product being called Autograder.io and the main deployment. I guess at the end of the day, my users (staff and students) don't need to know the product name, they just see it as CSAutograder.

Thanks again for your help.
Liam

**Comment by james-perretta** [comment]
@liamoreilly I see, a bit sloppy on my part throwing that change in on that commit, sorry for the confusion! Typically I would put configuration options like this in one of the other places that already has configurable constants, but I'm not sure how possible that is with the main index.html page since there isn't really any code that runs there in the same way as with a component. 

I'll go ahead and leave this open as a reminder to add a note to the docs on how to customize this.

**Comment by liamoreilly** [comment]
@james-perretta Not at all, I shouldn't have jumped to that conclusion. 

I don't think the change of title is a big issue, in the sense that we already have to change code files that are part of the repository when configuring autograder (as opposed to .env files that don't get committed). This mainly comes into play when upgrading as one has to be very careful with stashing changes, and then reapplying them. The change of title is a trivial change to manage. The complexity in upgrades comes from the range of files with changes that need stashing and reapplying. Keeping this small is key to making upgrades do-able. Just my thoughts.

**Comment by james-perretta** [comment]
@liamoreilly Agreed on the challenges of stashing/reapplying changes when upgrading--I've been bitten by that more recently than I'd care to admit...
I'll have to keep thinking about ways of making that process less error-prone, maybe a script or ideally some other solution that solves the problem at a more fundamental level.
